Examples/Tests/PML/analysis_pml_psatd_rz.py

```diff
@@ -0,0 +1,49 @@
+#! /usr/bin/env python
+
+# Copyright 2021 David Grote
+#
+#
+# This file is part of WarpX.
+#
+# License: BSD-3-Clause-LBNL
+
+"""
+This script tests the absorption of fields in the PML in RZ geometry.
+
+The input file inputs_particle_rz is used: it features an electron
+moving radially that launches a pulse. This scripts runs until
+most of the pulse escapes the radial boundary. If the PML fails,
+the pulse will remain with in the domain.
+"""
+import sys
+import numpy as np
+import yt
+yt.funcs.mylog.setLevel(0)
+sys.path.insert(1, '../../../../warpx/Regression/Checksum/')
+import checksumAPI
+
+# Open plotfile specified in command line
+filename = sys.argv[1]
+ds = yt.load( filename )
+
+# yt 4.0+ has rounding issues with our domain data:
+# RuntimeError: yt attempted to read outside the boundaries
+#               of a non-periodic domain along dimension 0.
+if 'force_periodicity' in dir(ds): ds.force_periodicity()
+
+# Check that the field is low enough
+ad0 = ds.covering_grid(level=0, left_edge=ds.domain_left_edge, dims=ds.domain_dimensions)
+Ex_array = ad0['boxlib', 'Ex'].to_ndarray()
+Ez_array = ad0['boxlib', 'Ez'].to_ndarray()
+max_Ex = np.abs(Ex_array).max()
+max_Ez = np.abs(Ez_array).max()
+print( f'max Ex = {max_Ex}' )
+print( f'max Ez = {max_Ez}' )
+max_Efield = max(max_Ex, max_Ez)
+
+tolerance_abs = 2.
+print('tolerance_abs: ' + str(tolerance_abs))
+assert max_Efield < tolerance_abs
```

Comment:
Another small question: How was this absolute tolerance of `2.` chosen? In other words, how do we know that a maximum electric field lower than `2.` is "low enough"? Maybe we could add some kind of comment above these lines to clarify the rationale behind this `2.`, so that, if the assert fails in future PRs, there is some guidance for the PR's author on why that `2.` should remain a `2.` (or close to it), for instance.